plugins/aea-ledger-ethereum/aea_ledger_ethereum/ethereum.py

```diff
@@ -61,10 +64,182 @@
 _ABI = "abi"
 _BYTECODE = "bytecode"
 
+MAX_GAS_FAST = 1500
+
+# How many blocks to consider for priority fee estimation
+FEE_HISTORY_BLOCKS = 10
+
+# Which percentile of effective priority fees to include
+FEE_HISTORY_PERCENTILE = 5
+
+# Which base fee to trigger priority fee estimation at (GWEI)
+PRIORITY_FEE_ESTIMATION_TRIGGER = 100
+
+# Returned if above trigger is not met (GWEI)
+DEFAULT_PRIORITY_FEE = 3
+
+# In case something goes wrong fall back to this estimate
+FALLBACK_ESTIMATE = {
+    "maxFeePerGas": to_wei(20, "gwei"),
+    "maxPriorityFeePerGas": to_wei(DEFAULT_PRIORITY_FEE, "gwei"),  # GWEI
+    "baseFee": None
+}
+
+PRIORITY_FEE_INCREASE_BOUNDARY = 200  # percentage
+
+
+DEFAULT_EIP1559_STRATEGY = {
+    "max_gas_fast": MAX_GAS_FAST,
+    "fee_history_blocks": FEE_HISTORY_BLOCKS,
+    "fee_history_percentile": FEE_HISTORY_PERCENTILE,
+    "priority_fee_estimation_trigger": PRIORITY_FEE_ESTIMATION_TRIGGER,
+    "default_priority_fee": DEFAULT_PRIORITY_FEE,
```

Comment:
```suggestion
    "priority_fee_estimation_trigger": to_wei(PRIORITY_FEE_ESTIMATION_TRIGGER, "gwei"),
    "default_priority_fee": to_wei(DEFAULT_PRIORITY_FEE, "gwei"),
```
Change to gwei! We should keep all input params in wei and all outputs in wei..
change accordingly below.